### ISSUE TITLE
states: can't assign style values to states

### ISSUE THREAD

**Issue by childsb** [maqetta/maqetta#1839]
1) create a new HTML file
2) add a button, add a calendar
3) add a new state, PLEASE_WORK
4) select the calendar and then select state PLEASE_WORK
5) go to background color, change the style target to "ELEMENT.STYLE" and change the background color
6) switch to the default state.  notice that the background color stays to what you set in PLEASE_WORK state
7) If you change the background color again, it will also change in the PLEASE_WORK state.

in other words, you can't set style properties on individual states.

**Comment by JonFerraiolo** [comment]
Did you click the tiny checkbox at the bottom of the cascade that says "apply to current state only"?

**Comment by childsb** [comment]
no, i dont see that in FF. let me send a screen grab.

**Comment by childsb** [comment]
![PP](http://i42.tinypic.com/314cv9d.jpg)

**Comment by childsb** [comment]
thats both FF and Safari, did i miss it?

**Comment by JonFerraiolo** [comment]
I must have broken this with my rewrite of the States palette. Assigning to me.
